jobs/delorean/misc/github-events/github-events.yaml

```diff
@@ -4,6 +4,13 @@
     display-name: 'GitHub Events Monitor'
     project-type: pipeline
     concurrent: true
+    triggers:
+    - raw:
+        xml: |
```

Comment:
Couldn't find a better way to add this trigger - open to suggestions

Reply:
Nope, no better suggestions. If it works it works. Will verify now.